### ISSUE TITLE
Command line version parameter not working

### ISSUE THREAD

**Issue by BackOrder** [Shoes3/shoes3#65]
```
./Shoes.app/Contents/MacOS/shoes -v
cat: /Users/ian/Downloads/Shoes.app/Contents/MacOS/VERSION.txt: No such file or directory
```

**Comment by ccoupe** [comment]
a 'grep -R VERSION.txt *' would show that I wanted to get rid of it. I missed one.

**Comment by BackOrder** [comment]
https://github.com/Shoes3/shoes3/blob/master/platform/mac/shoes#L21

**Comment by BackOrder** [comment]
Is this fix part of 3.2.21? Let's tag the milestone to keep our books in order.
